lib/ansible/plugins/inventory/aws_ec2.py

```diff
@@ -39,6 +39,10 @@
               - To use tags as hostnames use the syntax tag:Name=Value to use the hostname Name_Value, or tag:Name to use the value of the Name tag.
           type: list
           default: []
+        keyed_groups:
```

Comment:
this should be in the 'constructed' fragment

Reply:
yep, it's here: https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/doc_fragments/constructed.py#L26